README.rst

```diff
@@ -5,12 +5,34 @@ This is a command-line interface for interacting with connexions content. The to
 
 This software requires:
 
- - Python >= 3.5
- - JRE >= 6
+- Python >= 3.5
+- JRE >= 6
+
+Install
+=======
+
+1. Install `git`
+1. Install `python3` (on OSX you can run `brew install python3`)
+1. Install `virtualenv`
+1. Download the source `git clone https://github.com/Connexions/nebuchadnezzar`
+1. `cd /path/to/nebuchadnezzar`
+1. Initialize the python virtual environment:
+  1. `virtualenv ./venv/ --python=python3.5`
+  1. `source ./venv/bin/activate`
+  1. `python setup.py develop`
```

Comment:
Much of this can be condensed to:
1. Install python3
   - Download from python.org
   - Or on OS X run `brew install python3`
1. `pip3 install nebuchadnezzar`
1. `neb --help` to verify the application is installed

What you have here is a development install.